sdk/purview/purview-datamap-rest/src/outputModels.ts

```diff
@@ -1387,10 +1300,8 @@ export interface AtlasRelationshipDefOutput {
    * The difference being in the lifecycles of the container and its children.
    * In the COMPOSITION case, the children cannot exist without the container.
    * For AGGREGATION, the life cycles of the container and children are totally independent.
-   *
-   * Possible values: "ASSOCIATION", "AGGREGATION", "COMPOSITION"
    */
-  relationshipCategory?: string;
+  relationshipCategory?: RelationshipCategoryOutput;
```

Comment:
why this is changed from extensible enum into fixed enum?

Reply:
need use latest main to refresh